locust/main.py

```diff
@@ -268,6 +269,24 @@ def parse_options():
         help="Enables the auto tuning ramping feature for finding highest stable client count. NOTE having ramp enabled will add some more overhead for additional stats gathering"
     )
 
+    # if want to use integration test feature
+    parser.add_option(
+        '--integration',
+        action='store_true',
+        dest='integration',
+        default=False,
+        help='Run the test as integration test without web UI'
+    )
+
+    # set number of repetition for integration test feature
+    parser.add_option(
+        '-t','--repetition',
+        action='store_true',
+        dest='repetition',
+        default=False,
+        help='Set number of repetition for integration test, without this option test will be run once. Only used together with --integration'
```

Comment:
Repetition has 3 levels of priority:
1) if -t is set, this one will be taken for all test cases, despite of what repetition value set in a test case.
2) if -t is not set,  the one set under test case will be used
3) if no repetition variable is set in the testcase, default 1 time repetition will be used.